ts/components/wallet/card/FooterRow.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * A component for the bottom row of the
+ * credit card component, which either
+ * contains the "last usage" string or a
+ * bottom padding in those cases where
+ * the "last usage" part is not required
+ */
+import { Text } from "native-base";
+import * as React from "react";
+import { Col, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import ROUTES from "../../../navigation/routes";
+import { Dispatch } from "../../../store/actions/types";
+import { selectCardForDetails } from "../../../store/actions/wallet/cards";
+import variables from "../../../theme/variables";
+import { CreditCard } from "../../../types/CreditCard";
+import ActionIcon from "./ActionIcon";
+import { CreditCardStyles } from "./style";
+
+type ReduxMappedProps = Readonly<{
+  selectCard: (item: CreditCard) => void;
+}>;
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  item: CreditCard;
+  showMsg?: boolean;
+}>;
+
+type Props = OwnProps & ReduxMappedProps;
+
+class FooterRow extends React.Component<Props> {
+  public static defaultProps = {
+    showMsg: true
+  };
+
+  public render() {
+    const { navigate } = this.props.navigation;
+    const { item } = this.props;
+    if (this.props.showMsg) {
+      // show "last usage" row
+      return (
+        <Row
+          style={CreditCardStyles.rowStyle}
+          size={6}
+          {...{
```

Comment:
Is this usefull?

Reply:
Just create a file `react-native-easy-grid.d.ts` inside `/ts/@types` with this content

```
import { GestureResponderEvent } from "react-native";

declare module "react-native-easy-grid" {
  export interface RowProps extends ViewProperties {
    onPress?: (event: GestureResponderEvent) => void;
  }
}
```

In this way you are extending the definition and you can use onPress. Let me know it it is ok.